### PR TITLE
feat: add favicon

### PR DESCRIPTION
**Pull request by nhsz**
This PR adds the correct favicon and updates `PageMetadata` component with it

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -15,6 +16,11 @@ export default function App({ Component, pageProps }: AppProps) {
     <ChakraProvider theme={theme}>
       <MDXProvider components={MDXComponents}>
         <Layout>
+          <Head>
+            <meta name='viewport' content='width=device-width, initial-scale=1' />
+            <link rel='icon' type='image/x-icon' href='/favicon.ico' />
+          </Head>
+
```
Comment (wackerow) [inline src/pages/_app.tsx:23]: @nhsz Do we need this here if we're already placing this within the PageMetadata component which is then used on every page?

Comment (corwintines) [reply src/pages/_app.tsx:23]: I think this can be removed. Tested without it, like you said, this is handled in the `PageMetaData` component

**Review by corwintines — CHANGES_REQUESTED** [review]
Just above comment, good to go other than that

File: src/pages/_app.tsx (hunk 1 above)
Comment (nhsz) [reply src/pages/_app.tsx]: Makes sense, updated

**Review by corwintines — APPROVED** [review]

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { ChakraProvider } from '@chakra-ui/react';
 import { AppProps } from 'next/app';
+import Head from 'next/head';
```
Comment (wackerow) [inline src/pages/_app.tsx:3]:
```suggestion
```

**Pull request merged**